help/C/Help_ch_Transactions.xml

```diff
@@ -43,8 +43,9 @@ Translators:
     </listitem>
 
     <listitem>
-      <para>Importing transactions either from a <acronym>QIF</acronym>/<acronym>OFX</acronym> file or via
-      "<acronym>OFX</acronym> on-line banking".</para>
+      <para>Importing transactions either from an <acronym>OFX</acronym>,
+        <acronym>QFX</acronym>, <acronym>QIF</acronym> or <acronym>CSV</acronym>
+        file or via "<acronym>OFX</acronym> on-line banking".</para>
     </listitem>
```

Comment:
Hi Geert,
Re new style window frame:
Do you mean crop out the titlebar from screen images?
I think the window title is important, especially when it contains variable info, like the Register window, which includes the account in the title.
Whenever I see screenshots without the titlebar, I think they look disembodied. It doesn't overly worry me that there are different style windows in the manuals. I thought I'd mention it though. The new debian style actually looks a lot like Windows 10 now so that will feel familiar to those users, and the white background titlebar doesn't stand out like a colored background would.
On balance, I'd prefer to keep the titlebar.

Re imports: As we don't yet have good import documentation, I'll add the types you mention so at least they are listed somewhere other than the menus.